tests/5_container_runtime.sh

```diff
@@ -1032,18 +1032,40 @@ check_5_26() {
 }
 
 check_5_27() {
-  if [ -z "$containers" ]; then
-    return
-  fi
-
   local id="5.27"
-  local desc="Ensure that Podman commands always make use of the latest version of their image (Manual)"
+  local desc="Ensure that Podman commands always make use of the latest version of their image (Automatic)"
   local remediation="You should use proper version pinning mechanisms (the <latest> tag which is assigned by default is still vulnerable to caching attacks) to avoid extracting cached older versions. Version pinning mechanisms should be used for base images, packages, and entire images. You can customize version pinning rules according to your requirements."
   local remediationImpact="None."
   local check="$id - $desc"
   starttestjson "$id" "$desc"
 
-  info -c "$check"
+  fail=0
+  newer_images=""
+  for img in $images; do
+    imgFullName=$(podman inspect "$img" --format '{{ index .RepoTags 0 }}' | tr -d '[]')
+    imgTag=$(echo "${imgFullName}" | cut -f 2 -d ':')
+    imgUrl="docker://$(echo "${imgFullName}" | cut -f 1 -d ':')"
+    latestTag=$(skopeo list-tags "${imgUrl}" | jq -r '[.Tags[] | select(. != "latest")] | max')
```

Comment:
Are you sure `|max ` is right?  That's going to compare numeric values alphabetically isn't it?

Reply:
```bash
$ echo '["v1","v10","v15","v100"]' | jq -r '. | sort'
[
  "v1",
  "v10",
  "v100",
  "v15"
]
```

Reply:
Something like this would work, but it assumes images are tagged with version-number-like things :disappointed: 

`skopeo list-tags '${imgUrl}' | jq -r '.Tags[] | select(. != "latest")' | sort -V | tail -1`

I don't have a good solution here, so maybe it's not worth fixing?  I won't block on it either way.

Reply:
Should we introduce a user defined variable for the jq statement which would override a default value? Something like this:
```bash
jQuery=${JQUERY_FROM_USER:-jq -r '[.Tags[] | select(. != "latest")] | max'}
latestTag=$(skopeo list-tags "${imgUrl}" | "${jQuery}")
```

Reply:
oh! That's actually a pretty good idea.  Though since it's executing stuff for a purpose, maybe use that in the name to help users.  Something like, `latest_jquery_cmd` (I'm not the best at naming these things).  I like the idea in general though, since this is something that will probably never exactly fit every tagging arrangement by default.

Reply:
Well, ain't that easy as I thought - if possible at all.
Storing code (and not data) inside a variable is in general a bad idea. See [here](http://mywiki.wooledge.org/BashFAQ/050). Made some trials like this:

```bash
local jQuery=${JQUERY_5_27:-jq -r \'[.Tags[] | select(. != \"latest\")] | max\'}
...
latestTag=$(skopeo list-tags "${imgUrl}" | ${jQuery})
```

But this will always lead to unix quoting issues like this one, no matter what quoting/escaping is used:

```bash
Section A - Check results
jq: error: syntax error, unexpected INVALID_CHARACTER, expecting $end (Unix shell quoting issues?) at <top-level>, line 1:
'[.Tags[]
jq: 1 compile error
```

Also tried to store the jq command as a bash array which basically works. However, how should a user override an array?
Any ideas from your side?
If not, I'd suggest to leave it as it is and merge the PR.